modular_zubbers/code/datums/components/datums/riding_mob.dm

```diff
@@ -0,0 +1,63 @@
+#define DEFAULT_IN_HAND_OFFSET_X 3
+#define DEFAULT_IN_HAND_OFFSET_Y 0
+
+/datum/component/riding/creature/human
+	var/obj/item/bodypart/used_hand
+
+/datum/component/riding/creature/human/Initialize(mob/living/riding_mob, force, ride_check_flags, potion_boost)
+	. = ..()
+	var/mob/living/carbon/human/human_parent = parent
```

Comment:
rename this to steed/ridden/hunan_ridden or something of the like- it's not 100% clear that this is the person being ridden

Reply:
will do